draft-ietf-jsonpath-base.md

```diff
@@ -1108,17 +1108,23 @@ Since bracket notation is more general than dot notation, it is used to construc
 the syntax of Normalized Paths is restricted so that there is one and only one way of representing any
 given Normalized Path.
 
+Single quotes are used to delimit string member names. This makes it possible to use Normalized Paths
+as strings in JSON documents (delimited with double quotes) without needing to escape the contents of
+the strings.
+
 ~~~~ abnf
 normalized-path           = root-selector *(S normal-index-selector)
 normal-index-selector     = "[" S (normal-quoted-member-name / normal-element-index) S "]"
 normal-quoted-member-name = %x27 *normal-single-quoted %x27 ; 'string'
 normal-single-quoted      = normal-unescaped /
                             ESC normal-escapable
-normal-unescaped          = %x20-26 /                       ; omit control codes
+normal-unescaped          = %x20-21 /                       ; omit control codes
+                            %x23-26 /                       ; omit "
                             %x28-5B /                       ; omit '
                             %x5D-7E                         ; omit \
```

Comment:
I would comment on the missing `/`, but I think we'll need to get rid of the following line anyway (or change it considerably).

Reply:
What's interesting about `/`?

Reply:
There is none in the current ABNF between %x5D-7E and %xA0-FF :-)

Reply:
Ah, gotcha. I was so focussed on the character ranges, I couldn't see the missing `/`. Fixed.